lib/cdo/delete_accounts_helper.rb

```diff
@@ -128,24 +127,11 @@ def clean_and_destroy_pd_content(user_id)
     end
 
     unless pd_enrollment_ids.empty?
-      workshop_material_order_ids += Pd::WorkshopMaterialOrder.where(pd_enrollment_id: pd_enrollment_ids).pluck(:id)
```

Comment:
should we leave this in until we actually drop the db table? and would we be obligated to delete the info from thhe gsheet archive in the future?

Reply:
As a record of a business transaction we should be able to keep this, so I'm not worried about removing it from the hard-delete logic.